Docs/Workflows.md

```diff
@@ -54,24 +54,15 @@ jobs:
     name: Build and Test
     uses: Lombiq/GitHub-Actions/.github/workflows/build-and-test-dotnet.yml@dev
     with:
-      machine-types: "[\"ubuntu-22.04\", \"windows-2022\"]"
+      machine-types: "['ubuntu-22.04', 'windows-2022']"
       timeout-minutes: 10
 ```
 
 ## Spelling workflow
 
-Checks for spelling mistakes in a repository using the [Check Spelling](https://github.com/marketplace/actions/check-spelling) GitHub Action. There are 3 configuration files for filtering false positives:
+Checks for spelling mistakes in a repository using the [Check Spelling](https://github.com/marketplace/actions/check-spelling) GitHub Action, proxied by the `spelling` action in this repository, which has [its own documentation](../.github/actions/spelling/action.yml) describing the configuration options and contribution guidelines.
```

Comment:
Also link to the MD docs of the action, since that's necessary if you want to fully utilize the workflow.

Reply:
Oh, I actually meant to link the docs here, not the action, but why not both?